python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -0,0 +1,855 @@
+                ),
+                inplace=True,
+            )
+            ret["timediff"] = ret["timestamp"].diff()
+
+            # calculate derivatives - acceleration
+            den = ret["timediff"]
+            ret["demandAcceleration"] = ret["demandVelocity"].diff().div(den, axis=0)
+            ret["actualAcceleration"] = ret["actualVelocity"].diff().div(den, axis=0)
+            logging.info(
+                f"Retrieved {axis} {len(ret.index)} rows. "
+                f"Ranges: {ret.demandPosition.min():+.2f} .. "
+                f"{ret.demandPosition.max():+.2f}"
+                f", {ret.actualPosition.min():+.2f} .. "
+                f"{ret.actualPosition.max():+.2f}"
+            )
+            return ret
+
+        self.elevations = await find_axis("elevation")
+        self.azimuths = await find_axis("azimuth")
+
+    async def find_applicable_times(self) -> None:
+        """Find start and end times of any axis movents. Uses self.elevations
+        and self.azimuths data retrieved in find_az_el method.
+
+        Fills self.intervals with start and stop times usefull for fitting.
+        """
+        assert self.azimuths is not None
+        assert self.elevations is not None
+
+        ret = []
+
+        movements = pd.concat(
+            [
+                self.elevations[
+                    (abs(self.elevations.actualVelocity) > self.actual_velocity_limit)
+                    | (abs(self.elevations.demandVelocity) > self.demand_velocity_limit)
+                ],
+                self.azimuths[
+                    (abs(self.azimuths.actualVelocity) > self.actual_velocity_limit)
+                    | (abs(self.azimuths.demandVelocity) > self.demand_velocity_limit)
+                ],
+            ]
+        )
+
+        movements.sort_index(inplace=True)
+        movements["timediff"] = movements.index.to_series().diff()
+
+        start = None
+        end = None
+        for index, row in movements[
+            (movements.timediff > np.timedelta64(1, "s")) | (movements.timediff is None)
+        ].iterrows():
+            end = index - row["timediff"]
+            if start is not None and (end - start) > pd.Timedelta(seconds=0.5):
+                if self.was_raised(start, end):
+                    ret.append(
+                        [
+                            start,
+                            end,
+                            True,
+                        ]
+                    )
+                else:
+                    logging.warning(
+                        f"Interval {start.isot} - {end.isot} "
+                        "mirror was not raised, ignoring."
+                    )
+            start = index
+
+        # if no movement, use full range - this is for short times usefulla ss
+        # tests
+        if len(ret) == 0:
+            ret = [
+                [
+                    movements.index[0],
+                    movements.index[-1],
+                    True,
```

Comment:
why do you create this "use" column if it's always True? Can't we just remove it?

Reply:
It was originally used to filter out intervals. Dropped now.